doc/security/specs/ownership_transfer/_index.md

```diff
@@ -23,14 +23,26 @@ Implements transition from `OWNED` to `UNOWNED` state. The device must be in
 ### Ownership Transfer (or Ownership Assignment) {#ownership-transfer-top}
 
 Implements transition from `UNOWNED` to `OWNED` stage. The rest of this document
-refers to this functionality as Ownership Transfer. The following keys are
-provisioned as part of this flow:
+refers to this functionality as Ownership Transfer.
 
-Key        | Description
----------- | --------------------------------------------------------
-CODE_SIGN  | Used to verify the Silicon Owner first bootloader stage.
-UNLOCK     | Used to authenticate the Unlock Ownership command.
-NEXT_OWNER | Used to authenticate the Ownership Transfer command.
+## Owner Keys {#owner-keys}
+
+The following keys are provisioned as part of this flow.
+
+### CODE_SIGN
+
+*   **Description**: Used to verify the Silicon Owner first bootloader stage.
+*   **Key Type**: RSA-3072 public key with exponent 3 or F4.
```

Comment:
Interesting choice of algorithm. Why isn't this ECC, too? I guess it's quite small compared to the image payload, and verify is faster? (Is including justification for the algorithm choice a good idea?)

Reply:
RSA signature verification (using small exponents) is faster than ECDSA signature verification, while the opposite is true for signature and key generation.

Reply:
I added a note on the rationale for using RSA for code verification versus ECC for command authentication. It boils down to what @alphan  and @cdgori  mentioned in this thread.